mappings/net/minecraft/client/particle/ParticleFootstep.mapping

```diff
@@ -10,8 +10,19 @@ CLASS none/bnc net/minecraft/client/particle/ParticleFootstep
 			ARG 6 velocityY
 			ARG 7 velocityZ
 			ARG 8 extra
+	FIELD L maxAge_ I
+	FIELD M textureManager Lnone/bxf;
+	FIELD a TEX Lnone/kp;
+	FIELD b age_ I
```

Comment:
Some particles seem to define their own, identical age/maxAge fields so I've named them age_/maxAge_. No idea if there's any better name.